src/lib_cxx/data/parser/cxx/utilityClang.cpp

```diff
@@ -68,6 +68,8 @@ AccessKind utility::convertAccessSpecifier(clang::AccessSpecifier access)
 		return ACCESS_PRIVATE;
 	case clang::AS_none:
 		return ACCESS_NONE;
+	default:
+		return ACCESS_UNKNOWN;
```

Comment:
I just checked the rest of the code base. It would be fine to return `ACCESS_NONE` here as a default.

Reply:
done